Frontend/CO.CDP.OrganisationApp/Pages/Shared/_FlashMessage.cshtml

```diff
@@ -13,4 +13,18 @@
             <p class="govuk-notification-banner__heading">@(tempDataService.Get<string>(FlashMessageTypes.Success))</p>
         </div>
     </div>
+}
+
+@if (tempDataService.Peek<string>(FlashMessageTypes.Important) != null)
+{
+    <div class="govuk-notification-banner" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
```

Comment:
Spacing is a bit tight above this message. Could you add the same top margin from the notification banner?

<img width="812" alt="{7ECEBC74-77CE-44CE-AAA1-9EB70FD7C6FB}" src="https://github.com/user-attachments/assets/1aedd123-b11d-44ee-94c7-f9a342f4b953">

Reply:
```suggestion
    <div class="govuk-notification-banner govuk-!-margin-top-8" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
```

Reply:
Good ideas, done both, thanks for the review.